content/PIV/Guides/Device_setup.adoc

```diff
@@ -7,19 +7,27 @@ of the PUK and Management keys in a secure location.
 
 === Prerequisites
 * a PIV enabled YubiKey
-* the link:/yubico-piv-tool[yubico-piv-tool] or link:/yubikey-piv-manager[YubiKey PIV Manager] software
-
-=== Using YubiKey PIV Manager for device setup
-YubiKey PIV Manager will automatically prompt you to initialize the device by
-setting the PIN, PUK, and Management Keys when it detects a YubiKey which is
-using the default values. Follow the on-screen instructions to set new values
-for your YubiKey. The YubiKey PIV Manager presents you with an option to only
-set the PIN and not set a PUK or Management Key. This can be more convenient,
-but also less secure. Also note that this scheme is not compatible with other
-tools. Do not select this option if you are planning on changing any of these
-keys using tools other than the YubiKey PIV Manager (this includes
-yubico-piv-tool). For more information on this mode, see
-link:/yubikey-piv-manager/PIN_and_Management_Key.html[PIN and Management Key].
+* the link:/yubico-piv-tool[yubico-piv-tool] or link:/yubikey-manager-qt[YubiKey Manager] software
+
+=== Using YubiKey Manager for device setup
+YubiKey Manager allows you to change the PIN, PUK and Management Key.
+
+    Applications > PIV > Configure PINs
+
+The Management Key can be protected with the PIN, meaning that it's saved on
+the device in a location only readable with the PIN. This lets the user access the
+key management features while only having to remember the PIN.
+
+The link:/yubikey-manager[YubiKey Manager CLI] can also be used for device setup.
+
+  $ ykman piv change-pin
+  $ ykman piv change-puk
+  $ ykman piv change-management-key
+
+It also allows you to generate a random management key and store it on the device,
+protected with the PIN.
+
+  $ ykman piv change-management-key --generate --protect
 
 === Using yubico-piv-tool for device setup
```

Comment:
Still references yubico-piv-tool.

Reply:
Yes, do you think I should drop those references as well?